scripts/build/electron-sign-app-darwin.sh

```diff
@@ -87,13 +87,11 @@ IFS=$SAVEIFS
 # its components have been signed
 sign_file "$ARGV_APPLICATION"
 
-# Verify signature
-codesign \
-  --verify \
-  --deep \
-  --display \
-  --verbose=4 "$ARGV_APPLICATION"
-
+# spctl manages the security assessment policy subsystem.
+# This subsystem maintains and evaluates rules that determine whether the system
+# allows the installation, execution, and other operations on files on the system.
+# We use this tool to simulate whether the OS would accept our code
+# signature at the moment of installing/runnings.
```

Comment:
👍  Wdyt about changing the wording to "[...] moment of installation / execution" though? Runnings sounds a bit weird in this context :)